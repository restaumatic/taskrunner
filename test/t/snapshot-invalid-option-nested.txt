# Test similar to `snapshot-invalid-option`, but the task is nested.
# Tests (quite indirectly) that snapshot command goes to the correct command pipe (nested, not toplevel).
taskrunner -n nested-task bash -e -c '
  snapshot --junk-option
'
