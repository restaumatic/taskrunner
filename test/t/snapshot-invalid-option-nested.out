-- output:
[nested-task] error | snapshot: Invalid option or missing argument: --junk-option
-- exit code: 1
